.github/build.yml

```
@@ -0,0 +1,58 @@
+name: Build and test
+on: push
+jobs:
+  install-dependencies:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '16.x'
+      - name: Install Yarn
+        run: npm install -g yarn
+      - name: Setup Nodejs with yarn caching
+        uses: actions/setup-node@v2
+        with:
+          node-version: '16.x'
+          cache: yarn
+      - run: pushd frontend
+      - name: Install dependencies
+        run: yarn
+      - run: popd
+  test-files:
```

```suggestion

  test-files:
```
You have a space under this section, maybe before it too?